cypress.config.ts

```
@@ -1,10 +1,10 @@
-import { defineConfig } from 'cypress';
-import fs from 'node:fs';
-import codeCoverageTask from '@cypress/code-coverage/task';
-import dotenv from 'dotenv';
+import { defineConfig } from "cypress";
```

ESLint

[error] Replace `"cypress"` with `'cypress'`

(prettier/prettier)